sklearn/utils/extmath.py

```diff
@@ -374,6 +374,124 @@ def randomized_svd(M, n_components, *, n_oversamples=10, n_iter='auto',
         return U[:, :n_components], s[:n_components], Vt[:n_components, :]
 
 
+@_deprecate_positional_args
+def randomized_eigsh(M, n_components, *, n_oversamples=10, n_iter='auto',
```

Comment:
This needs adding to `doc/modules/classes.rst`

Reply:
ok